addons/fluxcd/definitions/helm-release-def.cue

```diff
@@ -177,7 +177,7 @@ template: {
 
 		git?: {
 			// +usage=The Git reference to checkout and monitor for changes, defaults to master branch
-			branch: string
+			branch?: *"master" | string
```

Comment:
Please do not use `?` and default value at the same time.

If you use `?` to mark the value as an optional field, then the default value will be ingored.

You can set the value to: `branch: *"master" | string` and make it optional in ui schema.

Reply:
Thanks.